### DISCUSSION TITLE
3 monitors possible?

### DISCUSSION THREAD

**Discussion by eiger3970** [AvaotaSBC/.github#4] [General]
Hello, I see the [specifications](https://github.com/AvaotaSBC/Avaota-A1), however I'm unsure if I could run 3 monitors?
I see Display interfaces:  including RGB interface, MIPI DSI interface, DP interface, HDMI2.0 interface, LVDS interface, MIPI DBI interface

If that's not enough for 3 monitors, perhaps the USB interfaces could be used? Like a USB-C.
If I read USB interfaces correctly, there's 3 right? USB interfaces: including USB OTG, USB3.0 OTG, USB2.0 HOST

**Comment by SamulKyull** [comment]
If the display shows the same image, then three monitors can be supported. However, if they show different images, only two monitors are possible, or three monitors with two displaying the same image. beside I didn't include SPI TFT. SPI screens are independently rendered through software and are not included.

**Reply by eiger3970** [reply]
Sorry, I haven't ordered the ARM SBC yet. I see 2 HDMI ports, but can't figure out the other possible display ports?

**Reply by SamulKyull** [reply]
There 1 HDMI, 1 DP, 1 LVDS/RGB/MIPI Combo FPC for LCD Panel Module, and 1 Onboard SPI TFT
![image](https://github.com/AvaotaSBC/.github/assets/166363804/b1ca8cb6-0a8f-45f1-86fa-0ba412670d5a)

**Comment by eiger3970** [comment]
Ok, seems risky as the 3 monitors will need an HDMI adapter to connect to the MIPI/RGB/LVDS or SPI. Then whether the OS will handle it? Any suggestions welcome.

**Reply by SamulKyull** [reply]
Currently, only the Android OS supports triple screens. Due to the ongoing reconstruction of DRM drivers for Linux, this feature will be available in future releases.

**Reply by eiger3970** [reply]
Armbian OS surely supports triple screens?

**Reply by SamulKyull** [reply]
Currently, only the Android, https://docs.avaota.fun/en-US/android/
